server/controllers/userAuthController.js

```diff
@@ -1,109 +1,18 @@
-import userModel from '../models/userAuth';
+// import userModel from '../models/userAuth';
 import Helper from '../helpers/helper';
+import Db from '../db';
 
 const userAuthController = {
+  createUser: (req, res) => new Db().addUser(req.body, res),
```

Comment:
its also not recommended to keep passing down res. The controller should be the one handling receiving a request & returning a response. Instead let Db().addUser return something from the db and then let the controller return failure (400 | 500) or success (201)